debezium-connector-mysql/src/test/java/io/debezium/relational/ddl/DdlChangesTest.java

```diff
@@ -1,97 +0,0 @@
-/*
```

Comment:
Why is this test completely removed?

Reply:
All its methods test `DdlChanges#groupEventsByDatabase()` which isn't used anywhere else than in this test.